third_party/glad/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright 2013 The Flutter Authors. All rights reserved.
+
```

Comment:
Note that if we didn't need a wrapper script, we also wouldn't need this, and could just use src/third_party and put the GN file in the buildroot repo, just as with GLFW.

Reply:
I observed that GN can't run python modules in `python -m scripts/glad` mode, so generate.py seems to be necessary.